packages/go_bootstrap.rb

```diff
@@ -22,16 +22,16 @@ class Go_bootstrap < Package
 
   def self.build
     FileUtils.cd('src') do
-      system "TMPDIR=/usr/local/tmp ./make.bash"
+      system "TMPDIR=#{CREW_PREFIX}tmp ./make.bash"
     end
   end
 
   def self.install
-    dest = "#{CREW_DEST_DIR}/usr/local/lib/go_bootstrap"
+    dest = "#{CREW_DEST_DIR}#{CREW_LIB_PREFIX}/go_bootstrap"
```

Comment:
`#{CREW_DEST_LIB_PREFIX}`?

Reply:
You know?  This might be one of those cases where we don't need to abstract the path.  Please test to find out.